test/Microsoft.Azure.Devices.Edge.Test/SetupFixture.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using NUnit.Framework;
+    using Serilog;
+    using Serilog.Events;
+
+    [SetUpFixture]
+    public class SetupFixture
+    {
+        readonly IEdgeDaemon daemon;
+        readonly IotHub iotHub;
+        EdgeDevice device;
+
+        public SetupFixture()
+        {
+            this.daemon = Platform.CreateEdgeDaemon(Context.Current.InstallerPath);
+            this.iotHub = new IotHub(
+                Context.Current.ConnectionString,
+                Context.Current.EventHubEndpoint,
```

Comment:
So, context is global?

Reply:
Yes, following the pattern of NUnit's TestContext and other types of config in our codebase.